sample-api-server/api_home/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:xenial
+RUN apt-get update
+
+RUN apt-get install --no-install-recommends -y apache2 libapache2-mod-python software-properties-common 
+RUN add-apt-repository ppa:certbot/certbot
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends python-certbot-apache 
+RUN a2enmod ssl cgi rewrite proxy_http
+
+## Clean out all the apt-stuff to reduce the image size
+RUN apt-get clean
+RUN rm -rf /var/cache/apt 
+RUN rm -rf /var/lib/dpkg/info/* 
+RUN rm -rf /var/lib/apt/lists/* 
+
+CMD ["/usr/sbin/apache2ctl", "-D", "FOREGROUND"]
```

Comment:
Is there any unblessed open source software being used while creating the docker image?

Reply:
Actually this applies to the whole pull request.

Reply:
There is no third party software included in the project.  
All third party software is pulled in by the end user as part of the build.  
Ubuntu base Docker image   
Apache HTTPD  + mod-python  
LetsEncrypt